Automattic/jetpack
# VideoPress: Add "learn how" chapters modal

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: Add how learn chapters modal

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to Block Editor
* Create/edit a video block
* Open block settings (sidebar)
* Confirm you see the `How Learn` link

<img width="497" alt="Screen Shot 2022-11-16 at 09 13 15" src="https://user-images.githubusercontent.com/77539/202177926-88c329d7-121a-4160-b91d-f124f1798b35.png">

* Confirm it opens the dialog by clicking on it
* Confirm the dialog is ok
<img width="990" alt="image" src="https://user-images.githubusercontent.com/77539/202177833-41e20a09-b8d6-4b9e-9608-8b3b5fc30cec.png">
* Close the dialog
